tensorboard/uploader/proto/write_service.proto

```diff
@@ -2,6 +2,10 @@ syntax = "proto3";
 
 package tensorboard.service;
 
+// TODO(bileschi): write_service.proto imports export_service.proto for the
```

Comment:
Ok, reading comments from @wchargin on #3227 it's true that removing the Experiment from export_service.proto will require internal python changes at sync time, but I'm not sure it's worth trying to avoid that anyway.  If we did, we'd have to do a much more involved series of steps (create a copy of the proto under a different temporary name, then update the python code to use the copy, then move the actual proto message to the new location, then update the python code to use that, and finally remove the temporary proto).

But at any rate, I was wrong to say it's "very straightforward" so ok to defer (but then let's revert the addition of `experiment.proto` in this PR until we do that change).

Reply:
I had thought experiment.proto has been reverted.  My bad.  Removed.